access.go

```diff
@@ -363,27 +365,49 @@ func (s *Server) handleRefreshTokenRequest(w *Response, r *http.Request) *Access
 func (s *Server) handlePasswordRequest(w *Response, r *http.Request) *AccessRequest {
 	// get client authentication
 	auth := GetClientAuth(w, r, s.Config.AllowClientSecretInParams)
-	if auth == nil {
+	if auth == nil  {
 		return nil
 	}
 
 	// generate access token
 	ret := &AccessRequest{
 		Type:            PASSWORD,
 		Username:        r.Form.Get("username"),
-		Password:        r.Form.Get("password"),
```

Comment:
the function here says it handle a password request, but the password is removed?

Reply:
ah thanks cove, 
i have fixed this